src/dp_transformers/arguments.py

```diff
@@ -4,7 +4,7 @@
 from transformers import TrainingArguments as HfTrainingArguments
 from transformers import IntervalStrategy, logging
 from dataclasses import dataclass, field
-from datasets import tqdm_utils
+from datasets.utils.logging import disable_progress_bar

```

Comment:
It is worth discussion versions of HF for our builds and being consistent. Should we update or should I revert this back to the old version?

Reply:
IMHO make it consistent with other examples, i.e., for now revert it so that the workflow succeeds. When we update versions, we update versions for all examples